winbuild/README.md

```diff
@@ -56,7 +56,7 @@ Open a Visual Studio Command prompt:
  [type] Native of Cross platform build.  This type of command prompt may not
  exist in all Visual Studio versions.
 
- See also: [Set the Path and Environment Variables for Command-Line Builds](https://msdn.microsoft.com/en-us/library/f2ccy3wt.aspx)
+ See also: [Set the Path and Environment Variables for Command-Line Builds](https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?redirectedfrom=MSDN&view=msvc-170)

```

Comment:
I think `https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line` should work without having to mention direct and explicit view in the URL

Reply:
Done! https://github.com/curl/curl/pull/8597/commits/3a2a6241d815a2d0c3d95e664257f85bc2d049d1